.travis.yml

```diff
@@ -2,27 +2,30 @@ language: go
 go:
 - 1.13.x
 
+env:
+  global:
+  - KREW_VERSION=v0.3.2
+  - GO111MODULE=on
+
 before_install:
   # install kubectl
   - sudo curl -fsSL -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.13.1/bin/linux/amd64/kubectl
   - sudo chmod +x /usr/bin/kubectl
 
   # install krew (temporarily hardcode version)
-  # TODO(ahmetb) pin this to a version via a variable
   - (
       cd "$(mktemp -d)" &&
-      curl -fsSLO "https://github.com/kubernetes-sigs/krew/releases/download/v0.3.1/krew.{tar.gz,yaml}" &&
+      curl -fsSLO "https://github.com/kubernetes-sigs/krew/releases/download/${KREW_VERSION}/krew.{tar.gz,yaml}" &&
       tar zxvf krew.tar.gz &&
       ./krew-linux_amd64 install --manifest=krew.yaml --archive=krew.tar.gz
     )
 
-  # TODO(ahmetb) pin this to a version via a variable
-  - env GOBIN=$HOME/bin go get sigs.k8s.io/krew/cmd/validate-krew-manifest
+  - env GOBIN=$HOME/bin go get sigs.k8s.io/krew/cmd/validate-krew-manifest@${KREW_VERSION}
```

Comment:
since when @ syntax started to work on go-gets on binaries lol
https://github.com/golang/go/issues/29415

I have a suspicion that this might be modifying our local `go.mod` file, and therefore causing a `-dirty` build?

Reply:
d'oh, we don't have a go.mod here.